src/lib/components/lines-and-dots/workflow-error.svelte

```diff
@@ -29,7 +29,7 @@
       return 'WorkflowTaskHeartbeatError';
     }
 
-    return toWorkflowTaskFailureReadable(cause);
+    return toWorkflowTaskFailureReadable(cause || 'Unspecified');
```

Comment:
Why not just have `toWorkflowTaskFailureReadable` do this if `cause` is `undefined`?

Reply:
Good call. Added unit test for it